### ISSUE TITLE
bigobj not found with MSYS Makefiles

### ISSUE THREAD

**Issue by dhebbeker** [doxygen/doxygen#7664]
Compiling doxygen with MSYS Makefiles I get an error, that `bigobj` can not be found. This seems natural, as I do not compile with MSVC but with GCC.

Using the following change allowed me to continue the build process:

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index a9545d01..15baaa60 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,11 +65,10 @@ if (WIN32)
     if (NOT CMAKE_GENERATOR MATCHES "MinGW Makefiles")
         if (NOT ICONV_DIR)
           set(ICONV_DIR "${CMAKE_SOURCE_DIR}/winbuild")
         endif()
         set(CMAKE_REQUIRED_DEFINITIONS "-DLIBICONV_STATIC")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /bigobj") # needed for language.cpp on 64bit
         add_definitions(-DLIBICONV_STATIC -D_CRT_SECURE_NO_WARNINGS)
     endif()
     if (CMAKE_GENERATOR MATCHES "NMake Makefiles")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
     endif()
```

I guess an additional condition for "MSYS Makfiles" should be added to `if (NOT CMAKE_GENERATOR MATCHES`?

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7666

(Testing would be appreciated)

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.18.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
